frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -154,6 +181,19 @@ const CreateVMWizardComponent: React.FC<CreateVMWizardComponentProps> = (props)
           },
         };
       }
+      if (nadsChanged) {
```

Comment:
even for very custom data this solution is not scalable: this PR fixes https://github.com/openshift/console/pull/6704 that. But still I added a TODO there, so we can come up with more efficient solution later for the 2 resources which use this approach.